library/openshift_package.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
```

Comment:
nit: Unless this is an ansible-ism generally docstrings occur after the license block.